src/Sulu/Component/Contact/Tests/Unit/SmartContent/ContactDataProviderTest.php

```diff
@@ -80,7 +80,7 @@ public function testGetDefaultParameter(): void
         $this->assertEquals([], $parameter);
     }
 
-    public function dataItemsDataProvider()
+    public static function dataItemsDataProvider(): array
```

Comment:
test method changes should be done on 2.5 branch if possible and non breaking changes

Here also a return type should be better specified for PHPStan